data/crac-creation/crac-creator-api/src/main/java/com/powsybl/openrao/data/craccreation/creator/api/parameters/CracCreationParameters.java

```diff
@@ -42,17 +42,17 @@ public interface ConfigLoader<E extends Extension<CracCreationParameters>> exten
     private String cracFactoryName = DEFAULT_CRAC_FACTORY_NAME;
 
     public enum MonitoredLineSide {
-        MONITOR_LINES_ON_LEFT_SIDE(Set.of(Side.LEFT)),
-        MONITOR_LINES_ON_RIGHT_SIDE(Set.of(Side.RIGHT)),
-        MONITOR_LINES_ON_BOTH_SIDES(Set.of(Side.LEFT, Side.RIGHT));
+        MONITOR_LINES_ON_LEFT_SIDE(Set.of(TwoSides.ONE)),
```

Comment:
To update with one two ?

Reply:
Yes, as we have decided, json for crac parameters will not have a new version, but nowadays, it will be able to read MONITOR_LINES_ON_LEFT/RIGHT_SIDE as MONITOR_LINES_ON_ONE/TWO_SIDE and it will always write MONITOR_LINES_ON_ONE/TWO_SIDE.